### PR TITLE
[Feature] Add components from @studiometa/js-toolkit

### PR DESCRIPTION
**Pull request by titouanmathis**
This PR sets up some configuration files for the project and adds components from the [@studiometa/js-toolkit](https://github.com/studiometa/js-toolkit) package.

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'no-underscore-dangle': 'off',
+  },
+  overrides: [
+    {
+      files: [ '**/*.spec.js', '**/spec.js', 'packages/tests/**/*.js' ],
+      extends: [ 'plugin:jest/recommended', 'plugin:jest/style' ],
+      rules: {
+        'max-classes-per-file': 'off',
+        'jest/no-test-callback': 'off',
+        'require-jsdoc': 'off',
```
Comment (KeziahMoselle) [inline .eslintrc.cjs:14]: Is there a reason to set this off ?

**Review by KeziahMoselle — APPROVED** [review]
LGTM

File: .eslintrc.cjs (hunk 1 above)
Comment (titouanmathis) [reply .eslintrc.cjs:14]: We do not need JS doc blocks in the test files 😉

**Review by notjb — APPROVED** [review]
🔥

**Comment by titouanmathis** [comment]
We will have to add the templates for each component, but we can add them later, I'll merge this.

**Pull request merged**